packages/i18nify-playground/src/sections/phoneNumber/phoneNumber-form.jsx

```diff
@@ -82,15 +84,15 @@ const PhoneNumberForm = ({
                     >
                       <div width="30px">+ {code}</div>
                       <img
-                        width="40px"
+                        width="28px"
```

Comment:
Why has this been changed ?

Reply:
40px looked really huge. have changed it back.